tests/unit/test_sensor.py

```diff
@@ -291,7 +293,7 @@ def test_noisy_rotated_accelerometer(noisy_rotated_accelerometer, example_plain_
         euler313_to_quaternions(*np.deg2rad([60, 60, 60]))
     )
     total_rotation = sensor_rotation @ cross_axis_sensitivity
-    rocket_rotation = Matrix.transformation(U[6:10])
+    rocket_rotation = Matrix.transformation(U[6:10]).transpose
```

Comment:
does the `transformation` method simply transpose the matrix? if so, shouldn't we just use U instead of transformation(U).transpose?

Reply:
The transformation method creates a rotation matrix from the quaternions